runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/JavaReadViaImpulse.java

```diff
@@ -1,176 +0,0 @@
-/*
```

Comment:
Just to make sure I'm understanding correctly: This transform was only used for portable pipelines, right? So it isn't needed anymore now that we execute bounded reads by wrapping them in an SDF?

Reply:
Yes

Reply:
Gotcha.

I already approved this, but consider it double-approved.